test/public/logs/overview.test.js

```diff
@@ -111,7 +111,7 @@ module.exports = () => {
         // Insert some other text into the filter
         await page.type('#titleFilterText', ' bogusbogusbogus');
         // Wait until there is no row with data left in the table
-        await page.waitForFunction(() => document.querySelector('#row1') === null, { timeout: 1500 });
+        await waitForTableLength(page, 0);
```

Comment:
I think here that you should have a dedicated function, because in this case loading state will also match.
You should have `waitForEmptyTable` and you specifically wait for `table tbody tr.empty-row`

Reply:
Done